colors/nord.vim

```diff
@@ -786,3 +804,56 @@ function! NordPalette() abort
   let ret["nord3_bright"] = s:nord3_gui_bright
   return ret
 endfunction
+
+" Treesitter
```

Comment:
The treesitter plugin actually does a lot of these links [already](https://github.com/nvim-treesitter/nvim-treesitter/blob/aa512ebca8704fbcecd05a9a4a6f82fc8f0c581f/plugin/nvim-treesitter.vim#L30) I think this it didn't before. But I don't know if there is any point if redoing them here? But i haven't looked to see if there is anything missing that should be added to nord...

Reply:
Then does this PR make any sense whatsoever?

Reply:
Most groups are already linked by default, but this doesn't mean that all of them are matching Nord's style.
I've added a couple of review comments to clarify things and clean up unnecessary definitions.